test/napi/native/src/lib.rs

```diff
@@ -1,3 +1,10 @@
 use neon::prelude::*;
 
-register_module!(_, { });
+register_module!(mut cx, {
+    let s = cx.string("Hello, World!");
+
+    println!("{}", s.value(&mut cx));
```

Comment:
#Fix: This can be tested a little better if I then create a new `JsString` from the value and assert that they are equal in the mocha test.